lib/feed.xml

```diff
@@ -1,26 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
-{% capture url_base %}{% if site.url %}{{ site.url | append: site.baseurl }}{% else %}{{ site.github.url }}{% endif %}{% endcapture %}
-<rss version="2.0"
-	xmlns:dc="http://purl.org/dc/elements/1.1/"
-	xmlns:atom="http://www.w3.org/2005/Atom"
-	>
-<channel>
+{% if site.url %}
+  {% assign url_base = site.url | append: site.baseurl %}
+{% else %}
+  {% assign url_base = site.github.url %}
+{% endif %}
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <generator uri="http://jekyllrb.com" version="{{ jekyll.version}}">Jekyll</generator>
+  <link href="{{ url_base }}{{ page.url }}" rel="self" type="application/atom+xml" />
+  <link href="{{ url_base }}/" rel="alternate" type="text/html"/>
+  <updated>{{ site.time | date_to_xmlschema }}</updated>
+  <id>{{ url_base }}/</id>
+
+  {% if site.name %}
     <title>{{ site.name }}</title>
-    <atom:link type="application/rss+xml" href="{{ url_base }}/feed/" rel="self"/>
-    <link>{{ url_base }}</link>
-    <pubDate>{{ site.time | date_to_rfc822 }}</pubDate>
-    <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
-    <generator>Jekyll v{{ jekyll.version }}</generator>
-    <description>{{ site.description }}</description>
-    {% for post in site.posts limit:site.rss_limit %}<item>
-        <title>{{ post.title }}</title>
-        <link>{{ url_base }}{{ post.url }}</link>
-        <pubDate>{{ post.date | date_to_rfc822 }}</pubDate>
-        <dc:creator>{{ post.author }}</dc:creator>
-        {% for tag in post.tags %}<category>{{ tag | xml_escape }}</category>
-        {% endfor %}{% for cat in post.categories %}<category>{{ cat | xml_escape }}</category>
-        {% endfor %}<guid isPermaLink="false">{{ post.id | xml_escape }}</guid>
-        <description><![CDATA[ {{ post.content }} ]]></description>
-    </item>{% endfor %}
-</channel>
-</rss>
+  {% endif %}
+
+  {% if site.description %}
+    <subtitle>{{ site.description }}</subtitle>
+  {% endif %}
+
+  {% if site.author %}
+    <author>
+      <name>{{ site.author }}</name>
+    </author>
+  {% endif %}
+
+  {% for post in site.posts %}
+    <entry>
+      <title>{{ post.title | xml_escape }}</title>
+      <link href="{{ url_base }}{{ post.url }}" rel="alternate" type="text/html" title="{{ post.title | xml_escape }}" />
+      <published>{{ post.date | date_to_xmlschema }}</published>
+      <updated>{{ post.date | date_to_xmlschema }}</updated>
+
+      <id>{{ url_base }}{{ post.id }}</id>
```

Comment:
Could we allow a site to override this, maybe with `guid` if present?

This will allow sites to move to Jekyll without breaking their existing feeds for users.

See #23